gh-pages-stub/public/js/search.js

```diff
@@ -0,0 +1,93 @@
+(function() {
+
+  function isBlank(str) {
+    return (!str || /^\s*$/.test(str));
+  } 
+
+  function displaySearchResults(results, store, searchTerm) {
+    var searchResults = document.getElementById('search-results');
+
+    if (results.length) {
+      var appendString = '';
+
+      for (var i = 0; i < results.length; i++) {
+        var item = store[results[i].ref];
+        if (isBlank(item.title)) {
+            item.title = "No title";
```

Comment:
is it good idea to modify store item by link here, method name displaySearchResults, saw i beleave that this method should not modify anything